### PR TITLE
redirect api reference on v-master to v1.6

### PR DESCRIPTION
**Pull request by ys2843**
## Description ##
Temporary fix for #18604

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:

### Changes ###
- [x] Temporarily redirect docs reference link on website from master to v1.6

## Comments ##
- Preview: http://ec2-34-219-134-42.us-west-2.compute.amazonaws.com/

### CODE REVIEW

**Comment by ys2843** [comment]
@mxnet-label-bot add [Website]

File: docs/static_site/src/.htaccess [hunk 1]
```diff
@@ -22,6 +22,14 @@ RewriteOptions AllowNoSlash
   
 </IfModule>
 
+# TODO temporary fix for issue #18604
+Redirect 302 /api/r/docs/api/R-package/build/mxnet-r-reference-manual.pdf https://s3.amazonaws.com/mxnet-prod/docs/R/mxnet-r-reference-manual.pdf
```
Comment (aaronmarkham) [inline docs/static_site/src/.htaccess:26]: There's a much more recent version of this and it's like 1000 pages long...

Comment (ys2843) [reply docs/static_site/src/.htaccess]: Updated to the newest version.

**Comment by sandeep-krishnamurthy** [comment]
I am bit concerned about redirecting someone visiting master to 1.6. I was hoping we could change default to latest stable as the solution for this fix. But, would like to hear your suggestion.

**Pull request merged**